app/views/petitions/show.html.erb

```diff
@@ -63,20 +63,11 @@
 
 <p class="signature-count"><%= number_with_delimiter(@petition.signature_count) %> <span>signatures</span></p>
 
-<%= render 'shared/about_petitions' %>
+<%= render partial: 'shared/about_petitions', locals: { variation: "about-petitions-light" } %>
```

Comment:
The default for `render` is now `:partial` so you can just use the following:

``` erb
<%= render 'shared/about_petitions', variation: 'about-petitions-light' %>
```

As you can see, there's no need to wrap the locals as `locals: {}`